03.API-EXAMPLE.md

```diff
@@ -521,4 +521,4 @@ Value of x is 17
 
 ## Further steps
 
-For further API description, please visit [API Reference page]({{ site.github.url }}/api-reference/) on [JerryScript home page]({{ site.github.url }}/).
+For further API description, please visit [API Reference page](https://samsung.github.io/jerryscript/api-reference/) on [JerryScript home page](https://samsung.github.io/jerryscript/).
```

Comment:
In the previous commit this line has been set manually, but the webpage-generator script set this line automatically.